pkgs/tools/networking/cloud-custodian/default.nix

```
@@ -0,0 +1,46 @@
+{ lib, buildPythonApplication, fetchPypi
+, argcomplete
+, boto3
+, botocore
+, certifi
+, dateutil
+, jsonpatch
+, jsonschema
+, pyyaml
+, tabulate
+, urllib3
+}:
+
+buildPythonApplication rec {
+  pname = "cloud-custodian";
+  version = "0.8.45.1";
+
+  src = fetchPypi {
+    pname = "c7n";
+    inherit version;
+    sha256 = "0c199gdmpm83xfghrbzp02xliyxiygsnx2fvb35j9qpf37wzzp3z";
+  };
+
+  propagatedBuildInputs = [
+    argcomplete
+    boto3
+    botocore
+    certifi
+    dateutil
+    jsonpatch
+    jsonschema
+    pyyaml
+    tabulate
+    urllib3
+  ];
+
+  # Requires tox, many packages, and network access
+  doCheck = false;
```

I would like to verify that it's working somewhat automatically. At least some measure that it won't be completely broken by certain dependency bumps
```suggestion
  checkPhase = ''
    $out/bin/custodian --help
  '';
```
otherwise LGTM

---

Applied.

@GrahamcOfBorg build cloud-custodian